kotlinx.fuzz.jazzer/src/main/kotlin/kotlinx/fuzz/jazzer/JazzerLauncher.kt

```diff
@@ -114,6 +114,9 @@ object JazzerLauncher {
 
         val atomicFinding = AtomicReference<Throwable>()
         FuzzTargetRunner.registerFatalFindingDeterminatorForJUnit { bytes, finding ->
+            System.err.println("Found finding: $finding")
+            System.err.println("Found bytes: ${bytes?.toHexString()}")
+            System.err.println("ReproducerPath: $reproducerPath:")
```

Comment:
Do you think we need those prints?

Reply:
I'm not sure, I left them after merge with #61. Probably we can remove them